rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
 . In the *Global Variables* view of the *Process Automation* module, click the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] in the table row of the global variable you want to delete.
 . Confirm the deletion.
 
-=== Create Credentials
+=== Manage Credentials
 
 Credentials store login data for applications or web services. They can be used with different processes. Passwords are encrypted.
 
+The following types of credentials can be created:
```

Comment:
A general suggestion for future writing - use active voice when possible; for example:

You can create the following types of credentials.